M2/Macaulay2/packages/undistributed-packages/RisaAsir.m2

```diff
@@ -67,7 +67,6 @@ Headline
   an interface to the Risa/Asir website and system
```

Comment:
This package I would like to flesh out, as Risa/Asir has a few very good algorithms